HT3.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+initial_text = r"""homEwork:
+	tHis iz your homeWork, copy these Text to variable. 
+
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE. 
+
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+"""
+
+# count whitespaces
+whitespace_cnt = re.findall('\s', initial_text)
+print('Number of whitespace characters is', len(whitespace_cnt))
+print('----------------')
+
+# correct misspelling
+text_replace_iz = initial_text.lower().replace(' iz ', ' is ')
+
+# split every paragraph
+split_by_enter = text_replace_iz.split('\n')
+# create an empty list for paragraphs
+paragraph_list = []
+# create an empty list for last sentence
+last_sent = []
+# split by dots
+for sentence in split_by_enter:
+    sentence = sentence.split('. ')

```

Comment:
TO REVIEW:
it's not a good practice to reuse name of the iterated entity in the code. If you want to use splited by words sentence, let's use split in the cycle definition, it's more readable and allows avoiding confusion during refactoring